dashboard/components/reports.py

```
@@ -0,0 +1,23 @@
+    
+    @staticmethod
+    def generate_json_report(data: Dict) -> str:
+        """Generate JSON report"""
+        report = {
+            'timestamp': datetime.now().isoformat(),
+            'data': data
+        }
+        return json.dumps(report, indent=2)
+    
+    @staticmethod
+    def generate_markdown_report(data: Dict) -> str:
```

Ruff (0.12.2)

Unused static method argument: `data`

(ARG004)